sceptre/resolvers/project_variables.py

```diff
@@ -4,6 +4,8 @@
 import yaml
 
 from sceptre.resolvers import Resolver
+from warnings import warn, simplefilter
+from colorama import Fore, init
```

Comment:
Could you move these imports up? The general rule we follow is:

```
python stdlib imports
<newline>
external library imports
<newline>
sceptre specific imports
```

So something like:
```
import os
import yaml
from warnings import warn, simplefilter

from colorama import Fore, init
  		  
from sceptre.resolvers import Resolver
```